vehicle_profiles/hyundai/ioniq-2017.json

```diff
@@ -3,13 +3,33 @@
   "init": "ATSP6;ATSH7E4;ATST96;",
   "pids": [
     {
-      "pid": "21057",
+      "pid": "2105",
       "parameters": [
         {
-          "name": "SOC_DISPLAY",
+          "name": "soc_display",
```

Comment:
Changing the name here will break things for people already using it, best to maintain the legacy name

Reply:
agreed, I more of a camelCase type but I'm more of a stability kinda guy... I'm already ingesting data using the capitalized entry

Reply:
Too bad there is no real standard for this. Maybe we should also add a friendly name too, to use in Home Assistant.
I reverted this backward incompatible change.